stellar/deploy-contract.js

```diff
@@ -289,7 +291,7 @@ async function createUpgradeAuths(contractAddress, newWasmHash, migrationData, c
     return Promise.all(
         [
             createAuthorizedFunc(contractAddress, 'upgrade', [nativeToScVal(newWasmHash)]),
-            createAuthorizedFunc(contractAddress, 'migrate', [nativeToScVal(migrationData)]),
+            migrationData ? createAuthorizedFunc(contractAddress, 'migrate', [nativeToScVal(migrationData)]) : null,
```

Comment:
Isn't the `migrate` method always called, regardless of if the migration data is empty?